.travis.yml

```diff
@@ -7,11 +7,10 @@ notifications:
   email: false
 node_js:
   - '10'
-
 script:
   - npm run bundle
-  - npm test 
-
+  - npm test
+  - npm run site
```

Comment:
would it make more sense to have `cd packages/d3fc-site && npm run site` here, rather than in the `package.json` ?

Reply:
Possibly; there's a few bad options here - I just picked one that was easy to test locally!